echo/server/tests/common.py

```
@@ -19,6 +19,7 @@
 
 
 def delete_project(project_id: str):
+    logger.debug(f"Deleting project: {project_id}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)